Code/GraphMol/CIPLabeler/CIPLabeler.h

```diff
@@ -20,6 +20,24 @@ class ROMol;
 
 namespace CIPLabeler {
 
+RDKIT_CIPLABELER_EXPORT bool decrementRemainingCallCountAndCheck();
+
```

Comment:
I'm not particularly fond of this being in the public API. 
It looks like it does need to be used across multiple files, so I'd suggest either putting it in a `detail` namespace to mark it as not-for-use in client code or moving it to a new header file, perhaps `CIPUtils.h`, which isn't installed on client systems (i.e. it isn't in the `rdkit_headers` section of `CMakeLists.txt`).

Reply:
I changed it to in a detail names as you suggested.